html/changelogs/geeves-shaft_miner_kit.yml

```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Shaft miners now start with a mining coin that can be redeemed at a mining vendor to get enhanced equipment, such as a mining drill or an upgraded kinetic accelerator, instead of spawning with a non-functional upgrade in a toolbox."
```

Comment:
```suggestion
  - rscadd: "Shaft miners now start with a mining coin that can be redeemed at a mining vendor to get enhanced equipment, such as a hand-held mining drill or an upgraded kinetic accelerator, instead of spawning with a non-functional upgrade in a toolbox."
```